server.js

```diff
@@ -4759,6 +4759,40 @@ cache(function(data, match, sendBadge, request) {
   sendBadge(format, badgeData);
 }));
 
+// homebrew integration
+camp.route(/^\/homebrew\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var package = match[1];
+  var format = match[2];
+  var apiUrl = 'http://braumeister.org/formula/' + package + '/version';
+
+  var badgeData = getBadgeData('homebrew', data);
+  request(apiUrl, { headers: { 'Accept': 'application/json' } }, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+    }
+    try {
+      var data = JSON.parse(buffer);
+      var version = data.stable;
+
+      badgeData.text[1] = 'v' + version;
+      if (version.indexOf('-') !== -1) {
+        badgeData.colorscheme = 'yellow';
+      } else if (version[0] === '0') {
+        badgeData.colorscheme = 'orange';
+      } else {
+        badgeData.colorscheme = 'blue';
```

Comment:
For your colors and version, I'd recommend using `var vdata = versionColor(version); badgeData.text[1] = vdata.version; badgeData.colorscheme = vdata.color;`. It ensures the version consistently starts with a `v`, and gives the same coloring scheme as the rest (which here closely matches what you have written).